### PR TITLE
brcm63xx: add USB status LED in 01_leds for ADP P.DG A4001N

### PR DESCRIPTION
**Pull request by DanCasterIt**
Add a status LED in 01_leds for USB 2-1 connected devices.

Signed-off-by: Daniele Castro <danielecastro@hotmail.it>

### CODE REVIEW

File: target/linux/bcm63xx/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -25,7 +25,8 @@ adb,pdg-a4001n-a-000-1a1-ax)
 	ucidef_set_led_netdev "lan" "LAN" "$model:green:internet" "eth0.1"
 	ucidef_set_led_netdev "wan" "WAN" "$model:green:adsl" "eth0.2"
 	ucidef_set_led_netdev "wlan0" "WIFI" "$model:green:wifi" "wlan0"
-	ucidef_set_led_usbdev "usb" "USB" "$model:green:service" "1-1"
+	ucidef_set_led_usbdev "usb1" "USB1" "$model:green:service" "1-1"
+	ucidef_set_led_usbdev "usb2" "USB2" "$model:red:service" "2-1"
```
Comment (adschm) [inline target/linux/bcm63xx/base-files/etc/board.d/01_leds:29]: Does this handling of green and red LEDs match vendor behavior?

Comment (DanCasterIt) [reply target/linux/bcm63xx/base-files/etc/board.d/01_leds:29]: This is the vendor manual:

Comment (DanCasterIt) [reply target/linux/bcm63xx/base-files/etc/board.d/01_leds:29]: Yes, it does

**Comment by ynezz** [comment]
This needs rebase and probably squashing of the commits as well.

**Comment by DanCasterIt** [comment]
> 
> 
> This needs rebase and probably squashing of the commits as well.

Problem solved

**Comment by Noltari** [comment]
Merged, thanks!
https://github.com/openwrt/openwrt/commit/9e7a0f45d8c4e8ccde67f80c24c7145be0130105

**Pull request closed without merging**